doc/luigi_patterns.rst

```diff
@@ -19,21 +19,94 @@ have some kind of periodical garbage collection clean it up.
 Triggering Many Tasks
```

Comment:
I separated the range aspect out of this pattern's original content; the wrapping pattern still kind of can stand on its own.